taxi/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    pass
```

Comment:
u can use admin.site.register instead )

Reply:
I wanted to follow only one style of register) But I will fix it